raremetalworker/4.13.7/Dockerfile

```diff
@@ -0,0 +1,46 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         raremetalworker
+# Software Version: 4.13.7
+# Description:      single variant analysis, generating summary statistics for gene level meta analyses
+# Website:          http://genome.sph.umich.edu/wiki/RAREMETALWORKER
+# Tags:             Genomics
+# Provides:         raremetalworker 4.13.7
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/raremetalworker 4.13.7/.
+# Pull Cmd:         docker pull biodckrdev/raremetalworker
+# Run Cmd:          docker run --rm -it biodckrdev/raremetalworker <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)